internal/definition/organization/schema.go

```diff
@@ -17,14 +17,16 @@ func newSchema() map[string]*schema.Schema {
 				Type:             schema.TypeString,
 				ValidateDiagFunc: check.Email,
 			},
-			Required: true,
+			Required:    true,
+			Description: "A list of email address that can be matched against existing users of the organization.",
 		},
 		"users": {
 			Type:     schema.TypeList,
 			Computed: true,
 			Elem: &schema.Schema{
 				Type: schema.TypeString,
 			},
+			Description: "Provides a list of user ids that match the emails provided returned in order of emails provided.",
```

Comment:
Suggestion: 

Description: "Provides a list of user IDs that match the emails provided. The user IDs are returned in the order of the provided emails.",